docs/users/troubleshooting.md

```diff
@@ -44,9 +44,8 @@ This means there is another webserver listening on the named port(s) and ddev ca
 
 To resolve this conflict, choose one of two methods:
 
-1. If you are using another local development environment (MAMP, WAMP, lando, etc.) that uses these ports, consider stopping it.
-2. Fix port conflicts by configuring your project to use different ports.
-3. Fix port conflicts by stopping the competing application.
+1. Fix port conflicts by configuring your project to use different ports.
```

Comment:
Got it - that's fine. The only reason I'm suggesting an update here is because these are numbered 1, 2, 3, but the headers/descriptions below list "Method 1" and "Method 2" which correspond to the above number 2 and 3, not 1 and 2. So it's slightly misleading when determining which steps to follow.